rust/utils/run-locally/src/main.rs

```diff
@@ -388,6 +375,24 @@ fn main() -> ExitCode {
     state.watchers.push(kathy_stderr);
     kathy.wait().unwrap();
 
+    for (i, validator_env) in validator_envs.iter().enumerate() {
```

Comment:
nit: update the comment above "Send half the kathy messages before starting the agents"